ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,185 @@
+import os
+
+from ..internal.logger import get_logger
+from ..constants import ENV_KEY
+from ..constants import VERSION_KEY
+
+log = get_logger(__name__)
+
+
+OTEL_UNIFIED_TAG_MAPPINGS = {
+    "deployment.environment": ENV_KEY,
+    "service.name": "service",
+    "service.version": VERSION_KEY,
+}
+
+
+def _remap_otel_log_level(otel_value):
+    """Remaps the otel log level to ddtrace log level"""
+    if otel_value == "debug":
+        return "True"
+    elif otel_value == "info":
+        return "False"
+    else:
+        log.warning(
+            "ddtrace does not support otel log level '%s'. setting ddtrace to log level info.",
+            otel_value,
+        )
+        return "False"
+
+
+def _remap_otel_propagators(otel_value):
+    """Remaps the otel propagators to ddtrace propagators"""
+    accepted_styles = []
+    for style in otel_value.split(","):
+        style = style.strip().lower()
+        if style in ["b3", "b3multi", "b3single", "datadog", "tracecontext", "none"]:
+            if style == "b3single":
+                style = "b3"
+            if style not in accepted_styles:
+                accepted_styles.append(style)
+        else:
+            log.warning("Following style not supported by ddtrace: %s.", style)
+    return ",".join(accepted_styles)
+
+
+def _remap_traces_sampler(otel_value):
+    """Remaps the otel trace sampler to ddtrace trace sampler"""
+    if otel_value == "always_on" or otel_value == "parentbased_always_on":
+        return "1.0"
+    elif otel_value == "always_off" or otel_value == "parentbased_always_off":
+        return "0.0"
+    elif otel_value == "traceidratio" or otel_value == "parentbased_traceidratio":
+        return os.environ.get("OTEL_TRACES_SAMPLER_ARG", "1")
+    else:
+        return otel_value
+
+
+def _remap_traces_exporter(otel_value):
+    """Remaps the otel trace exporter to ddtrace trace exporter"""
+    if otel_value != "none":
+        log.warning("An unrecognized trace exporter '%s' is being used; setting DD_TRACE_ENABLED to False.", otel_value)
+        return "False"
+    else:
+        return "True"
```

Comment:
According the specification all unrecognized exporters should be logged as warnings. It does not say we should disable tracing if we see an unrecognized exporter. My understanding is that if we detected an unsupported exporter (ex: otlp,zipkin,console) we do nothing and let tracing be enabled by default.

```suggestion
        log.warning("An unrecognized trace exporter '%s' is being used; setting DD_TRACE_ENABLED to False.", otel_value)
        return ""
    else:
        return "False"
```